package.json

```diff
@@ -31,17 +31,17 @@
     "husky": "^7.0.4",
     "mocha": "^9.2.0",
     "ts-mocha": "^9.0.2",
-    "typescript": "^4.2.4"
+    "typescript": "^5.2.2"
```

Comment:
This upgrade necessary as other security-related upgrades use advanced TS features (sigh)